### PR TITLE
Modify Html/Css

### PR DESCRIPTION
**Pull request by CELYNEBIRUNGI**
Updated HTML And  CSS .
Updated Readme file
Checked Eslint  errors
Added Javascript File

### CODE REVIEW

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @CELYNEBIRUNGI  😊,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request 🆘💀
You have submitted a project, that I see that you created the project by yourself no partner, so please can you make your partner commit to the project 😃 as this is pair programming activity, you should please contact student success at "support@microverse.zendesk.com" if you are having an issue reaching out to your partners.
Note: only student success can give you the green light to do the pair programming task as a solo but you also have to include a screenshot from student success allowing you to do so.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.you should please contact student success at "support@microverse.zendesk.com" if you are having an issue reaching out to your partners.
Note: only student success can give you the green light to do the pair programming task as a solo but you also have to include a screenshot from student success allowing you to do so.

File: index.html [hunk 1]
```diff
@@ -21,9 +21,9 @@
     <div i d="mobilemenu" class="mobile-menu-inactive">
     <div class="disabled-button"><a href="#"><img src="images/hamburger disable.png"></a></div>
     <ul class="mobile-menu-list">
-      <li><a id="button1" class="mobile-menu-links" href="#main">Portfolio</a></li>
-     <li><a id="button2" class="mobile-menu-links" href="#about-myself">About</a></li>
-      <li><a id="button3" class="mobile-menu-links" href="#form">Contact</a></li>
+    <li><a id="button1" class="mobile-menu-links" href="#main">Portfolio</a></li>
+    <li><a id="button2" class="mobile-menu-links" href="#about-myself">About</a></li>
+    <li><a id="button3" class="mobile-menu-links" href="#form">Contact</a></li>
```
Comment (m05e5) [inline index.html]: - [ ] As part of the requirements, you are supposed to display the mobile menu. please fix the bug and make your mobile menu appear when we click on the menu icon 
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @CELYNEBIRUNGI , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @CELYNEBIRUNGI, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Mobile menu ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**